### PR TITLE
Added the option to block analytics system

### PR DESCRIPTION
**Pull request by Failcookie**
Added the option to block analytics system (specifically Google Analytics, HubSpot and Lucky Orange) and have it be controlled by the user. User can set the type by using 
`->setBlockAnalytics(true)`. The default is false. References this issue #82. Note: This is my first pull request, so please let me know if I did something incorrect here.

### CODE REVIEW

File: src/Capture.php [hunk 1]
```diff
@@ -235,12 +242,16 @@ public function save($imageLocation, $deleteFileIfExists = true)
...
+		    $data['blockAnalytics'] = $this->blockAnalytics;
+	    }
+
         if ($deleteFileIfExists && file_exists($this->imageLocation) && is_writable($this->imageLocation)) {
             unlink($this->imageLocation);
         }
 
         $jobName = md5(json_encode($data));
-        $jobPath = $this->jobs->getLocation() . $jobName . '.js';
+        $jobPath = $this->jobs->getLocation() . rand(1, 1000) . '.js';
```
Comment (Failcookie) [inline src/Capture.php]: This is incorrect - I put that in there for testing only and failed to remove before pushing.

File: src/Capture.php [hunk 2]
```diff
@@ -89,13 +89,20 @@ class Capture
      */
     protected $userAgentString = '';
 
+	/**
```
Comment (peter-mw) [inline src/Capture.php]: @MASNathan can you remove this "Nitpick-CI" bot, its spamming my dashboard like crazy

Comment (MASNathan) [inline src/Capture.php]: @peter-mw Done!

File: templates/screen-capture.php [hunk 3]
```diff
@@ -14,6 +14,26 @@
...
+	}
+	//
+	if ((/track.hubspot.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
+	if ((/luckyorange\.com/gi).test(requestData['url'])){
+		request.abort();
+	}
+	// Lucky Orange Tracking
```
Comment (MASNathan) [inline templates/screen-capture.php]: Thank you for your submission @Failcookie 

I noticed that the comments indicating the tracking system that will be blocked are not fully correct, can you fix them please?

File: src/Capture.php (hunk 2 above)
Comment (peter-mw) [inline src/Capture.php]: thanks :)

File: templates/screen-capture.php (hunk 3 above)
Comment (Failcookie) [inline templates/screen-capture.php]: Thanks for the review, @MASNathan! I added comments for HubSpot and Google Analytics tracking. Was that what you were referring to?

Comment (MASNathan) [inline templates/screen-capture.php]: Yes thank you

**Pull request merged**